### ISSUE TITLE
Tell when posts are old

### ISSUE THREAD

**Issue by EmilMoe** [Hashnode/user-feedback#146]
**Name of the feature**
Mainly blogposts

**Describe the solution you'd like**
An information box on top if the blogpost is more than, maybe, 2 years old, to inform the reader the content might be outdated. But also an option for the author to validate the content and preferably a warning by email when the post is about to be marked as old.

**Describe alternatives you've considered**
—

**Additional context**
—

**Comment by BolajiAyodeji** [comment]
Hey @EmilMoe, thanks for raising this amazing issue. :+1: 
I'll relate this to my team and we'll get back to you.

**Comment by EmilMoe** [comment]
So what is the status? It seems that these are always forgotten.

**Comment by minecrawler** [comment]
I have given up on this tracker pretty much, because most issues are now closed by the GitHub bot for missing any kind of update after months. I don't know what the Hashnode team is even thinking or doing, but it's not very engaging. It's so sad to see all the good feedback just being closed and forgotten, in favor of changing the website's theme every year. For me, Hashnode is a pile of broken promises (I have a lot of stories to tell in this department), missing engagement (why ask for feedback if you just ignore it?) and I started wondering when they just close it all down for the lack of love for the community. Especially seeing a lot of Hashnode staff coming and going in such short intervals is no good sign, imho.

**Comment by EmilMoe** [comment]
> I have given up on this tracker pretty much, because most issues are now closed by the GitHub bot for missing any kind of update after months. I don't know what the Hashnode team is even thinking or doing, but it's not very engaging. It's so sad to see all the good feedback just being closed and forgotten, in favor of changing the website's theme every year. For me, Hashnode is a pile of broken promises (I have a lot of stories to tell in this department), missing engagement (why ask for feedback if you just ignore it?) and I started wondering when they just close it all down for the lack of commitment to features they announce big time, and the missing love for the community. Especially seeing a lot of Hashnode staff coming and going in such short intervals is no good sign, imho.

Yes unfortunately and we are almost back at design 1 (which was perfect in my opinion)

**Comment by fazlerocks** [comment]
The actions were not set correctly, due to which, some of the crucial issues were auto closed. However, we do review all the issues that users report and address the most critical ones first. @minecrawler 

**Coming back to this issue:**
We have not been able to come up with a good solution to identify thousands of discussions that are 2-3 years old. Some of the discussions are still relevant to the date and labelling them as old won't be the right approach.

**Comment by minecrawler** [comment]
@fazlerocks What I wrote still holds, as I can only judge the situation from the outside, and I am not happy. However, that's a discussion best hold somewhere else.

Back on topic: how about implementing a review-queue for authors and moderators, so people can get notifications about their own old threads regularly (every year?) and then can decide if something is old or might still be relevant. In addition, they might be able to update their content or bump the topic, if they think it deserves some new attention. There should be some kind of moderation in place, though. Bumping has a tradition of getting out of hand easily.
